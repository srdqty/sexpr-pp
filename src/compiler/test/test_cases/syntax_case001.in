(syntax (1 2 3))
(syntax a-symbol)
(syntax ())
