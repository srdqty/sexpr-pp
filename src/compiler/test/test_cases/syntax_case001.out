(Constant <syntax: (1 2 3)>)
(Constant <syntax: a-symbol>)
(Constant <syntax: ()>)
